ui/app/templates/components/namespace-picker.hbs

```diff
@@ -1,110 +1,84 @@
-{{#if (and (not this.accessibleNamespaces.length) this.inRootNamespace)}}
-  <div class="namespace-picker no-namespaces">
-    {{! Just yield the logo if they're in the root namespace and only have access to it }}
-    {{yield}}
-  </div>
-{{else}}
-  <div class="namespace-picker">
-    <BasicDropdown @horizontalPosition="auto-left" @verticalPosition="above" as |D|>
-      <D.Trigger
-        @htmlTag="button"
-        class="button is-transparent namespace-picker-trigger has-current-color"
-        data-test-namespace-toggle
-      >
-        {{yield}}
-        {{#if this.namespaceDisplay}}
-          <span class="namespace-name">{{this.namespaceDisplay}}</span>
-        {{else}}
-          <span class="namespace-name is-hidden-tablet">/ (Root)</span>
-        {{/if}}
-        <Chevron @direction="down" @class="has-text-white auto-width is-status-chevron" />
-      </D.Trigger>
-      <D.Content @defaultClass="namespace-picker-content">
+<div class="namespace-picker">
+  <BasicDropdown @horizontalPosition="left" @verticalPosition="above" as |D|>
+    <D.Trigger
+      @htmlTag="button"
+      class="button is-transparent namespace-picker-trigger has-current-color"
+      data-test-namespace-toggle
+    >
+      <div class="is-flex-center">
+        <Icon @name="org" />
+        <span class="namespace-name">{{this.namespaceDisplay}}</span>
+      </div>
+      <Icon @name="caret" />
+    </D.Trigger>
+    <D.Content @defaultClass="namespace-picker-content">
+      <div class="is-mobile level-left">
+        <h5 class="list-header">Current namespace</h5>
+      </div>
+      <div class="namespace-header-bar level is-mobile">
+        <div class="level-left">
+          <header>
+            <div class="level is-mobile namespace-link">
+              <span class="level-left" data-test-current-namespace>
+                {{if this.namespacePath (concat this.namespacePath "/") "root"}}
+              </span>
+            </div>
+          </header>
+        </div>
+      </div>
+      <div class="namespace-list {{if this.isAnimating 'animated-list'}}">
         <div class="is-mobile level-left">
           {{#unless this.isUserRootNamespace}}
             <NamespaceLink
               @targetNamespace={{or
                 (object-at (dec 2 this.menuLeaves.length) this.lastMenuLeaves)
                 this.auth.authData.userRootNamespace
               }}
-              @class="namespace-link is-current button is-ghost icon"
+              @class="namespace-link is-current button icon is-transparent"
```

Comment:
we have an `icon.is-blue` and a `button.is-transparent` class—not very consistent, I know. Maybe it's the ordering of the class names, what do you think about having the icon come last because it's really the button.is-transparent that is going to do some styling.

Reply:
Sure I can update that. Just for future thought, if we are interested in class name order on elements we should automate something to enforce that (alphabetical or something else) since everyone will put them in likely a random order.

Reply:
Something else to maybe keep in mind while we establish our updated styling patterns is that something like a `button-transparent` class that inherits the button styling and adds the transparency is probably more descriptive than having a `button` and `is-transparent` class if `is-transparent` only applies to buttons.

Reply:
yes, and yes. Totally agree.